ow_sim_tests/scripts/test_light_set_intensity.py

```diff
@@ -0,0 +1,143 @@
+    )
+
+  # Move the antenna pointing to the base_link for light intensity visualization
+  def test_03_pan_tilt_move_cartesian(self):
+    pan_tilt_move_cartesian_result = test_action(self,
+      'PanTiltMoveCartesian', owl_msgs.msg.PanTiltMoveCartesianAction,
+      owl_msgs.msg.PanTiltMoveCartesianGoal(
+        frame = 0,
+        point = Point(0, 0, 0)
+      ),
+      TEST_NAME, "test_03_pan_tilt_move_cartesian",
+    )
+  
+  def test_04_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'left',
+        intensity = 0.4
+      ),
+      TEST_NAME, 'test_04_light_set_intensity'
+    )
+
+  def test_05_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'left',
+        intensity = 0.6
+      ),
+      TEST_NAME, 'test_05_light_set_intensity'
+    )
+  
+  def test_06_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'left',
+        intensity = 0.8
+      ),
+      TEST_NAME, 'test_06_light_set_intensity'
+    )
+
+  def test_07_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'left',
+        intensity = 1.0
+      ),
+      TEST_NAME, 'test_07_light_set_intensity'
+    )
+
+  def test_08_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'right',
+        intensity = 0.4
+      ),
+      TEST_NAME, 'test_08_light_set_intensity'
+    )
+
+  def test_09_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'right',
+        intensity = 0.6
+      ),
+      TEST_NAME, 'test_09_light_set_intensity'
+    )
+  
+  def test_10_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'right',
+        intensity = 0.8
+      ),
+      TEST_NAME, 'test_10_light_set_intensity'
+    )
+
+  def test_11_light_set_intensity(self):
+    light_set_intensity_result = test_action(self,
+      'LightSetIntensity', owl_msgs.msg.LightSetIntensityAction,
+      owl_msgs.msg.LightSetIntensityGoal(
+        name = 'right',
+        intensity = 1.0
+      ),
+      TEST_NAME, 'test_11_light_set_intensity'
+    )
+    
```

Comment:
Please add two more tests here that both turn off the lights by setting their intensity values to `0.0`

Reply:
I'd like to see a delay between each intensity setting so that one can see it change.  The test ran so fast that all I observed was one, high, light intensity.